triggerbot/tree_watcher.py

```diff
@@ -96,15 +94,12 @@ def _get_jobs(self, branch, rev, hidden):
         return [job['ref_data_name'] for job in jobs
                 if not re.match('[a-z0-9]{12}', job['ref_data_name'])]
 
-
     def get_hidden_jobs(self, branch, rev):
         return self._get_jobs(branch, rev, True)
 
-
     def get_visible_jobs(self, branch, rev):
         return self._get_jobs(branch, rev, False)
 
-
```

Comment:
It's default for pep8 standard, 2 newlines between functions and 1 newline between instance methods.